src/mrunal/assignment7.java

```diff
@@ -0,0 +1,59 @@
+package mrunal;
+
+public class assignment7 {
+	void displayEven(int start, int end )
+	{
+		for(int i= start; i<=end;i++ )
+		{
+			if(i%2==0)
+			 System.out.println(i +" is even number");
+		}
+	}
+
+	void displayDivisiblebyFive(int start, int end )
+	{
+		for(int i= start; i<=end;i++ )
+		{
+			if(i%5==0)
+			 System.out.println(i +" is divisible by 5");
+		}
+	}
+	
+	void displayDivisiblebyFiveAndThree(int start, int end )
+	{
+		for(int i= start; i<=end;i++)
+		{
+			if(i%5==0 && i%3==0)
+			 System.out.println(i +" is divisible by 5 and 3");
+		}	
+		
+	}
+	
+	void displayDivisiblebySevenAndThirteen(int start, int end )
+	{
+		for(int i= start; i<=end;i++)
+
+		{
+			if((i%7==0) || (i%13==0))
+			 System.out.println(i +" is divisible by "  );
```

Comment:
Here, how u predict that number is divide by 7 or 13. here you need to 
if(number%7==0 )
				System.out.println(number + " is divisible by 7");
			else if(number%13==0)
				System.out.println(number + " is divisible by 13");